source/process/feature-release.md

```diff
@@ -19,6 +19,9 @@ Notes:
 
 Pre-work for the current release begins at the code complete date of the previous release. See "Code Complete" section below for details.
 
+1. **(Team)**
+    - Release Manager, Dev Leads, QA, and PMs to meet to discuss plans for the upcoming release, e.g. share designs, high level thoughts on what would be included along with dependencies.
```

Comment:
The meeting would probably normally happen a month prior to the feature complete.